specification/purview/resource-manager/Microsoft.Purview/stable/2021-12-01/purview.json

```diff
@@ -0,0 +1,2685 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "locations",
+            "in": "path",
+            "description": "Location of feature.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "featureRequest",
+            "in": "body",
+            "description": "Request body with feature names.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BatchFeatureRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/BatchFeatureStatus"
+            }
+          },
+          "default": {
+            "description": "An error response received from the provider.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Features_SubscriptionGet": {
+            "$ref": "./examples/Features_SubscriptionGet.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Purview/accounts/{accountName}/kafkaConfigurations/{kafkaConfigurationName}": {
+      "get": {
+        "tags": [
+          "KafkaConfiguration"
+        ],
+        "summary": "Gets the kafka configuration.",
+        "description": "Gets the kafka configuration for the account",
+        "operationId": "KafkaConfigurations_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/accountName"
+          },
+          {
+            "name": "kafkaConfigurationName",
+            "in": "path",
+            "description": "Name of kafka configuration.",
+            "required": true,
+            "type": "string",
+            "maxLength": 63,
+            "minLength": 3,
+            "pattern": "^[A-Za-z0-9]+(?:-[A-Za-z0-9]+)*$"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/KafkaConfiguration"
+            }
+          },
+          "default": {
+            "description": "An error response received from the provider.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "KafkaConfigurations_Get": {
+            "$ref": "./examples/KafkaConfigurations_Get.json"
+          }
+        }
+      },
+      "put": {
```

Comment:
Must contain 200 and 201 return codes.

refs: https://github.com/Azure/azure-openapi-validator/blob/main/docs/put-response-schema-description.md

Reply:
Will add 201

Reply:
Done